### PR TITLE
fix: dev issues on develop branch

### PR DESCRIPTION
**Pull request by didiermun**

### CODE REVIEW

File: src/redux/store.tsx [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
   reducer: {
-
-    auth: authReducer
-  }
-})
-
-export type RootState = ReturnType<typeof store.getState>
-
     auth: authReducer,
   },
```
Comment (stasgga) [inline src/redux/store.tsx:8]: This is an update provided by Elisa on slack but I did not find it on github when I pulled

**Review by nathanbengineer — DISMISSED** [review]

**Review by veritem — DISMISSED** [review]

**Review by twizelissa — DISMISSED** [review]

**Review by chanelle740 — DISMISSED** [review]

**Review by yvettegahamanyi — DISMISSED** [review]
Ready

**Review by chanelle740 — DISMISSED** [review]

**Review by Muhodari — DISMISSED** [review]

**Review by veritem — APPROVED** [review]

File: src/screens/BusesScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+export default function Buses() {
+  const pageSize = 7;
+  const linkCount = 3;
+  const users = data;
+  const search = '';
+  const pageData: IAdmin[] = [];
+  const currentPage = 1;
+  const currentLinkPage = 1;
+  let selected: number[] = [];
```
Comment (veritem) [inline src/screens/BusesScreen.tsx:58]: this should have been put into a useState. This is react bruh

**Review by veritem — CHANGES_REQUESTED** [review]

**Review by Yassin-hagenimana — DISMISSED** [review]

**Review by tuyishimejeandamour — DISMISSED** [review]

File: src/screens/BusesScreen.tsx (hunk 2 above)
Comment (didiermun) [reply src/screens/BusesScreen.tsx:58]: Yh, that is sth I will do in my next push

**Review by nathanbengineer — APPROVED** [review]
good to go

**Review by twizelissa — APPROVED** [review]

**Pull request closed without merging**